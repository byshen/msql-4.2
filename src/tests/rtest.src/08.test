# Check out the correct handling of money values

create table bad_money( v1 money)\p\g

insert into bad_money values (29.00)\p\g
insert into bad_money values (29.01)\p\g
insert into bad_money values (29.03)\p\g
insert into bad_money values (29.04)\p\g
insert into bad_money values (29.05)\p\g
insert into bad_money values (29.06)\p\g
insert into bad_money values (29.07)\p\g
insert into bad_money values (29.08)\p\g
insert into bad_money values (29.09)\p\g
insert into bad_money values (29.10)\p\g
insert into bad_money values (29.11)\p\g
insert into bad_money values (29.97)\p\g
insert into bad_money values (29.98)\p\g
insert into bad_money values (29.99)\p\g
insert into bad_money values (30.00)\p\g

insert into bad_money values (-30.04)\p\g
insert into bad_money values (-30.03)\p\g
insert into bad_money values (-30.02)\p\g
insert into bad_money values (-30.01)\p\g
insert into bad_money values (-30.00)\p\g
insert into bad_money values (-29.99)\p\g
insert into bad_money values (-29.98)\p\g
insert into bad_money values (-29.97)\p\g
insert into bad_money values (-29.96)\p\g
insert into bad_money values (-29.95)\p\g
insert into bad_money values (-29.94)\p\g
insert into bad_money values (-29.93)\p\g
insert into bad_money values (-29.92)\p\g
insert into bad_money values (-29.91)\p\g

select * from bad_money order by v1 \p\g
