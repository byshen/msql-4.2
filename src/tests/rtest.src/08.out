
Welcome to the miniSQL monitor.  Type \h for help.


mSQL >     -> 
Query buffer
------------

create table bad_money( v1 money)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL >     -> 
Query buffer
------------
insert into bad_money values (29.00)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (29.01)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (29.03)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (29.04)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (29.05)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (29.06)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (29.07)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (29.08)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (29.09)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (29.10)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (29.11)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (29.97)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (29.98)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (29.99)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (30.00)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL >     -> 
Query buffer
------------
insert into bad_money values (-30.04)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (-30.03)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (-30.02)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (-30.01)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (-30.00)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (-29.99)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (-29.98)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (-29.97)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (-29.96)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (-29.95)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (-29.94)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (-29.93)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (-29.92)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL > 
Query buffer
------------
insert into bad_money values (-29.91)
[continue]
    -> 
Query OK.  1 row(s) modified or retrieved.




mSQL >     -> 
Query buffer
------------
select * from bad_money order by v1 
[continue]
    -> 
Query OK.  29 row(s) modified or retrieved.

 +----------+
 | v1       |
 +----------+
 | -30.04   |
 | -30.03   |
 | -30.02   |
 | -30.01   |
 | -30.00   |
 | -29.99   |
 | -29.98   |
 | -29.97   |
 | -29.96   |
 | -29.95   |
 | -29.94   |
 | -29.93   |
 | -29.92   |
 | -29.91   |
 | 29.00    |
 | 29.01    |
 | 29.03    |
 | 29.04    |
 | 29.05    |
 | 29.06    |
 | 29.07    |
 | 29.08    |
 | 29.09    |
 | 29.10    |
 | 29.11    |
 | 29.97    |
 | 29.98    |
 | 29.99    |
 | 30.00    |
 +----------+



mSQL > 
Bye!

